src/dcqc/suites/suite_abc.py

```diff
@@ -292,3 +292,9 @@ def from_dict(cls, dictionary: SerializedObject) -> SuiteABC:
     def get_base_class(cls):
         """Retrieve base class."""
         return SuiteABC
+
+    def get_status(self, compute_ok: bool = True) -> SuiteStatus:
```

Comment:
Maybe I missed this, but when would you use `compute_ok`?

Reply:
this was copied over from the `get_status` method in the tests module and I forgot to remove `compute_ok`. removed in latest commit.